pkg/liqoctl/status/peer/peerinfo.go

```diff
@@ -0,0 +1,352 @@
+// Copyright 2019-2022 The Liqo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package statuspeer
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/utils/strings/slices"
+
+	discoveryv1alpha1 "github.com/liqotech/liqo/apis/discovery/v1alpha1"
+	liqoconsts "github.com/liqotech/liqo/pkg/consts"
+	"github.com/liqotech/liqo/pkg/liqoctl/output"
+	"github.com/liqotech/liqo/pkg/liqoctl/status"
+	"github.com/liqotech/liqo/pkg/liqoctl/status/utils/resources"
+	liqonetutils "github.com/liqotech/liqo/pkg/liqonet/utils"
+	liqoutils "github.com/liqotech/liqo/pkg/utils"
+	liqogetters "github.com/liqotech/liqo/pkg/utils/getters"
+	liqolabels "github.com/liqotech/liqo/pkg/utils/labels"
+	peeringconditionsutils "github.com/liqotech/liqo/pkg/utils/peeringConditions"
+	"github.com/liqotech/liqo/pkg/utils/slice"
+)
+
+// PeerInfoOptions contains the options for the PeerInfoChecker.
+type PeerInfoOptions struct {
+	RemoteClusterNames []string
+}
+
+// PeerInfoChecker implements the Check interface.
+// holds the information about the peered cluster.
+type PeerInfoChecker struct {
+	o                *status.Options
```

Comment:
```suggestion
	options                *status.Options
```

I would be consistent with the other occurrences.